server/api/support.py

```diff
@@ -66,6 +68,43 @@ def auth0_create_audit_admin(email: str) -> Optional[str]:
         raise error
 
 
+@api.route("/support/elections/active", methods=["GET"])
+@restrict_access_support
+def list_active_elections():
+    elections = (
+        Election.query.filter(
+            Election.id.in_(
+                ActivityLogRecord.query.filter(
+                    ActivityLogRecord.timestamp
+                    > datetime.now(timezone.utc) - timedelta(days=14)
+                )
+                .with_entities(
+                    ActivityLogRecord.info["base"]["election_id"].as_string()
+                )
+                .subquery()
```

Comment:
Nice! I was curious to see how you're identifying "active" audits. This seems like a solid method, conceptually.

I'll be curious to see if it slows down when there's lots of activity, and retrieving 14 days worth of records potentially becomes a slow operation. Do we have any sense of this from past years, how expensive it might be to retrieve that much activity?

Reply:
That's a good question, though since it never gets returned back from the DB server but is just used as a subquery, I'm hoping it's not a huge issue. Still worth a sanity check, which I can do locally

Reply:
Ok, I faked the current date being the height of audit season last year (mid-December) and the request took <50ms. I think this is looking ok!